src/Components/Artwork/GridItem.tsx

```diff
@@ -1,27 +1,44 @@
-import { Flex, Sans } from "@artsy/palette"
+import { color, Flex, Sans } from "@artsy/palette"
 import { GridItem_artwork } from "__generated__/GridItem_artwork.graphql"
 import { Mediator } from "Artsy/SystemContext"
 import { isFunction } from "lodash"
 import React from "react"
+import { LazyLoadImage } from "react-lazy-load-image-component"
 import { createFragmentContainer, graphql } from "react-relay"
 import { data as sd } from "sharify"
-import styled from "styled-components"
-import colors from "../../Assets/Colors"
+import styled, { css, keyframes } from "styled-components"
 import Metadata from "./Metadata"
 import SaveButton from "./Save"
 
-const Image = styled.img`
+/**
+ * The animation that's used for the background of an image while it's loading
+ * in.
+ */
+const pulse = keyframes`
+  0% { background-color: ${color("black10")}; }
+  50% { background-color: ${color("black5")}; }
+  100% { background-color: ${color("black10")}; }
+`
+
+const pulseAnimation = props =>
+  css`
+    ${pulse} 2s ease-in-out infinite;
+  `
+
+const Image = styled(LazyLoadImage)`
```

Comment:
Wondering, should this be configured so that it lazy loads below the fold content? Similar to how we trigger the "Recently Viewed" rail to appear before the user has scrolled to it. Seeing the image jump in in real time isn't too optimial.

Reply:
(Ah, I see the `todo` item in the description)

Reply:
The library offers a threshold that we can use to set the load distance. It's currently 100px below the fold. Fast scrollers are one case, slow phones are another. We can tweak the setting if needed. Might be worth standardizing on a distance below-the-fold loading threshold.

Reply:
Cool, I bet a couple hundred pixels would be sufficient.